deployment/helm/node-feature-discovery/templates/master.yaml

```
@@ -5,6 +5,10 @@ metadata:
   labels:
     {{- include "node-feature-discovery.labels" . | nindent 4 }}
     role: master
+{{- with .Values.master.deploymentAnnotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+{{- end }}
```

How about the simpler form we use below for pod annotations?

```suggestion
  annotations:
    {{- toYaml .Values.master.deploymentAnnotations | nindent 4 }}
```